SingularityUI/app/reducers/api/base.es6

```diff
@@ -13,7 +13,9 @@ export default function buildApiActionReducer(ActionGroup, initialData=[]) {
       case ActionGroup.ERROR:
         return _.extend({}, state, {
           isFetching: false,
-          error: action.error
+          error: action.error,
+          data: null,
```

Comment:
this may have unintended consequences for other reducers -- this should be an optional arg in `buildApiActionReducer()` if you truly need data to be set to null in the case of an error. i also wonder if `null` or `initialData` is a better choice here?